App/Sagas/Account/AccountSagas.ts

```diff
@@ -86,3 +87,16 @@ export function * refreshCafeSessions () {
     }
   }
 }
+
+export function * getSDKVersion () {
+  while (true) {
+    try {
+      yield take(getType(AccountActions.refreshSDKVersionRequest))
+      const v: string = yield call(version)
+      yield put(AccountActions.getSDKVersionSuccess(v))
+      break
```

Comment:
You don't need this break. If you keep it there, the saga will terminate for good, any future requests will be ignored. I realize there will probably not be any future requests, but if you're going to do this in the request/fulfill requests kind of pattern, might as well just support working for any request at any time.